integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/chat_generator.py

```diff
@@ -78,23 +88,27 @@ def __init__(
         constructor. Aside from model, three required parameters are `aws_access_key_id`, `aws_secret_access_key`,
         and `aws_region_name`.
 
-        :param model: The model to use for generation. The model must be available in Amazon Bedrock. The model has to
-        be specified in the format outlined in the Amazon Bedrock [documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids-arns.html).
+        :param model: The model to use for text generation. The model must be available in Amazon Bedrock and must be
```

Comment:
```suggestion
        :param model: The model to use for text generation. The model must be available in Amazon Bedrock and must 
```

_be_ is on the next line :)